lab.R

```diff
@@ -114,23 +129,33 @@ distinct(dup_demo)
 #' Use this function to create a dataframe `dataf_nodup` with the duplicate rows removed.  Do `dataf` and `dataf_nodup` have the same number of rows?  What does this tell you about duplicate entries in our dataset?  
 #' 
 #' 
-#' 
-# dataf_nodup = ???
+#' they do. is there a way to do a tibble.shape?
+# dataf_nodup = 2930
```

Comment:
`dim(dataf)` gives rows and columns; `nrow(dataf)` gives number of rows.  Uncomment this line for the automatic checker.